quic/s2n-quic-transport/src/stream/controller.rs

```diff
@@ -321,24 +320,34 @@ mod tests {
     use s2n_quic_core::varint::VarInt;
 
     impl Controller {
-        pub fn available_outgoing_stream_capacity(&self, stream_type: StreamType) -> VarInt {
+        pub fn available_local_initiated_stream_capacity(&self, stream_type: StreamType) -> VarInt {
+            match stream_type {
+                StreamType::Bidirectional => self.local_bidi_controller.available_stream_capacity(),
+                StreamType::Unidirectional => self.local_uni_controller.available_stream_capacity(),
+            }
+        }
+
+        pub fn remote_initiated_max_streams_latest_value(&self, stream_type: StreamType) -> VarInt {
             match stream_type {
-                StreamType::Bidirectional => self
-                    .bidi_controller
-                    .local_initiated
-                    .available_stream_capacity(),
-                StreamType::Unidirectional => self
-                    .uni_controller
-                    .local_initiated
-                    .available_stream_capacity(),
+                StreamType::Bidirectional => self.remote_bidi_controller.latest_limit(),
+                StreamType::Unidirectional => self.remote_uni_controller.latest_limit(),
             }
         }
 
-        pub fn max_streams_latest_value(&self, stream_type: StreamType) -> VarInt {
+        pub fn available_remote_intiated_stream_capacity(&self, stream_type: StreamType) -> VarInt {
```

Comment:
```suggestion
        pub fn available_remote_initiated_stream_capacity(&self, stream_type: StreamType) -> VarInt {
```

Reply:
This is only used by tests?

Reply:
yep

Reply:
Maybe mark it with `#[cfg(test)]` then, so if we do end up using it outside of tests that will prompt some additional scrutiny

Reply:
yep this and the two other methods are inside a test mod already

```
#[cfg(test)]
mod tests {
```

Reply:
oh yeah, GitHub hid that from me!